IBM/ibm-common-service-operator
# [4.2] olm based webhook on cs-operator

How to test:

1. `export QUAY_REGISTRY=quay.io/<your username>`
2. `export REGISTRY=quay.io/<your username>`
3. `make build-dev-image`
4. edit manager.yaml with image `quay.io/<your username>/common-service-operator-amd64:dev`
5. `make bundle-manifests && make build-bundle-image && make run-bundle`
6. Verify cs-operator pod starts up, no errors in logs, two services `webhook-service` and `ibm-common-service-operator-service` are created, both `vcommonservice.kb.io` and `moperandrequest.kb.io` are working.
